File: Data Structures & Algorithms/duplicate-integer/submission-1.py
class Solution:
    def hasDuplicate(self, nums: List[int]) -> bool:
        num_set = set()

        for i in range(len(nums)):
            if nums[i] not in num_set:
                num_set.add(nums[i])
            else:
                return True
        return False

        

File: Data Structures & Algorithms/concatenation-of-array/submission-1.cpp
class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
        vector<int> ans;
        int counter = 0;

        while(counter<=1){
            for(int num:nums){
                ans.push_back(num);
            }
            counter ++;
        }

        return ans;
        
    }
};